tests/accidental_tests.ts

```diff
@@ -338,7 +338,9 @@ function genAccidentals(): string[] {
   }
   // Spartan Sagittal multi-shaft accidentals
   for (let u = 0xe310; u <= 0xe335; u++) {
-    switch (u) {  // exclude unused smufls
+    switch (
+      u // exclude unused smufls
+    ) {
```

Comment:
This looks weird. Why was the comment moved into here?

Reply:
Can't we restore it to what it was before? Or insert a new line above the switch and put the comment there.

Reply:
It is done automatically by eslint